core/path/path_linux.odin

```diff
@@ -0,0 +1,7 @@
+package path
+
+
+SEPARATOR        :: '/';
+SEPARATOR_STRING :: "/";
+
+#compile_assert("Linux is not yet supported");
```

Comment:
`#assert`

Also this assert doesn't get hit. Moving it to the second line does have it get hit

Reply:
Ah, thanks! I'll push an edit tomorrow.

Also, that's super weird about the second line thing...

Maybe I should just remove `path_linux.odin`, since there's nothing really in it anyway, and that way it'll behave like any other unimplemented platform. I don't really know what would happen if you tried to use it like that though... Probably just tell you none of the things are defined

Reply:
> I don't really know what would happen if you tried to use it like that though... Probably just tell you none of the things are defined

Yes, which it still does with the file since it doesn't define `full`, `is_dir`, or `current`. Removing the file just adds `SEPARATOR` and `SEPARATOR_STRING` to that list (and obviously removes the `#assert` error message)